langchain/package.json

```diff
@@ -199,6 +199,9 @@
     "document_loaders/fs/csv.cjs",
     "document_loaders/fs/csv.js",
     "document_loaders/fs/csv.d.ts",
+    "document_loaders/fs/md.cjs",
```

Comment:
Consider adding `document_loaders/fs/md.ts` in `typedocOptions` found in `tsconfig.json`